notebooks/Sigma_test_Notebook.ipynb

```diff
@@ -86,18 +86,84 @@
+       "0                                 NaN                            NaN   \n",
+       "1                                 NaN                            NaN   \n",
+       "2                                 NaN                            NaN   \n",
+       "3  GZIP:/var/log/apt/history.log.1.gz  /var/log/apt/history.log.1.gz   \n",
+       "\n",
+       "                        packages       parser  \n",
+       "0                            NaN          NaN  \n",
+       "1                            NaN          NaN  \n",
+       "2                            NaN          NaN  \n",
+       "3  Install: zmap:amd64 (1.1.0-1)  apt_history  "
+      ],
+      "text/html": "<div>\n<style scoped>\n    .dataframe tbody tr th:only-of-type {\n        vertical-align: middle;\n    }\n\n    .dataframe tbody tr th {\n        vertical-align: top;\n    }\n\n    .dataframe thead th {\n        text-align: right;\n    }\n</style>\n<table border=\"1\" class=\"dataframe\">\n  <thead>\n    <tr style=\"text-align: right;\">\n      <th></th>\n      <th>message</th>\n      <th>timestamp</th>\n      <th>datetime</th>\n      <th>timestamp_desc</th>\n      <th>extra_field_1</th>\n      <th>command</th>\n      <th>data_type</th>\n      <th>display_name</th>\n      <th>filename</th>\n      <th>packages</th>\n      <th>parser</th>\n    </tr>\n  </thead>\n  <tbody>\n    <tr>\n      <th>0</th>\n      <td>A message</td>\n      <td>123456789</td>\n      <td>2015-07-24T19:01:01+00:00</td>\n      <td>Write time</td>\n      <td>foo</td>\n      <td>NaN</td>\n      <td>NaN</td>\n      <td>NaN</td>\n      <td>NaN</td>\n      <td>NaN</td>\n      <td>NaN</td>\n    </tr>\n    <tr>\n      <th>1</th>\n      <td>Another message</td>\n      <td>123456790</td>\n      <td>2015-07-24T19:01:02+00:00</td>\n      <td>Write time</td>\n      <td>bar</td>\n      <td>NaN</td>\n      <td>NaN</td>\n      <td>NaN</td>\n      <td>NaN</td>\n      <td>NaN</td>\n      <td>NaN</td>\n    </tr>\n    <tr>\n      <th>2</th>\n      <td>Yet more messages</td>\n      <td>123456791</td>\n      <td>2015-07-24T19:01:03+00:00</td>\n      <td>Write time</td>\n      <td>baz</td>\n      <td>NaN</td>\n      <td>NaN</td>\n      <td>NaN</td>\n      <td>NaN</td>\n      <td>NaN</td>\n      <td>NaN</td>\n    </tr>\n    <tr>\n      <th>3</th>\n      <td>Install: zmap:amd64 (1.1.0-1) [Commandline: ap...</td>\n      <td>123456791</td>\n      <td>2015-07-24T19:01:03+00:00</td>\n      <td>foo</td>\n      <td>NaN</td>\n      <td>Commandline: apt-get install zmap</td>\n      <td>apt:history:line</td>\n      <td>GZIP:/var/log/apt/history.log.1.gz</td>\n      <td>/var/log/apt/history.log.1.gz</td>\n      <td>Install: zmap:amd64 (1.1.0-1)</td>\n      <td>apt_history</td>\n    </tr>\n  </tbody>\n</table>\n</div>"
+     },
+     "metadata": {},
+     "execution_count": 10
+    }
+   ],
+   "source": [
+    "# Now we convert the CSV to a pandas dataframe\n",
```

Comment:
I also prefer to hve these comments as a markdown cell, rather than being a comment inside a code cell